extensions/wikia/Search/classes/IndexService/BacklinkCount.php

```diff
@@ -1,26 +1,28 @@
 <?php
 /**
  * Class definition for \Wikia\Search\Indexservice\BacklinkCount
+ *
  * @author relwell
  */
 namespace Wikia\Search\IndexService;
 /**
- * This class provides backlink info for a page  
+ * This class provides backlink info for a page
+ *
  * @author relwell
  * @package Search
  * @subpackage IndexService
  */
-class BacklinkCount extends AbstractService
-{
+class BacklinkCount extends AbstractService {
 	/**
 	 * Returns an array keyed for a solr schema field for backlinks
+	 *
 	 * @see \Wikia\Search\IndexService\AbstractService::execute()
 	 * @return array
 	 */
 	public function execute() {
-		wfProfileIn(__METHOD__);
-		$response = array( 'backlinks' => $this->getService()->getBacklinksCountFromPageId( $this->currentPageId ) );
-		wfProfileOut(__METHOD__);
+
+		$response = [ 'backlinks' => $this->getService()->getBacklinksCountFromPageId( $this->currentPageId ) ];
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$response". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)